material/static/material/js/materialize.forms.js

```diff
@@ -15,6 +15,12 @@
       .not('.material-ignore')
       .material_select()
 
+    // Tabs
```

Comment:
And at least this is not the right place for that code.

The problem with tabs happens in the frontend, not in form, so materialize.frontend.js could be better the place for this if fronted would have tabs.